### PR TITLE
Rename log-files

### PR DESCRIPTION
**Pull request by pheanex**
Rename log-files to contain application_name, component and version, so we can do better cost attribution

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -3,9 +3,8 @@ build_steps:
   cmd: |
     apt-get update
     apt-get install -q -y --no-install-recommends \
-      git \
-      python3.5 \
-      python3.5-dev \
+      python3.6 \
```
Comment (anton-ryzhov) [inline delivery.yaml]: Will it work with `python3` and `python3-dev` instead?

File: kube_log_watcher/templates/scalyr.json.jinja2 [hunk 2]
```diff
@@ -18,6 +18,7 @@
         {% for log in logs %}
             {
                 "path": "{{ log.path }}",
+                "rename_logfile": "application={{ log.attributes.application | quote_plus }}/component={{ log.attributes.component | quote_plus }}/version={{ log.attributes.version | quote_plus }}",
```
Comment (anton-ryzhov) [inline kube_log_watcher/templates/scalyr.json.jinja2]: How about adding of some well-known prefix to simplify parsing? Something like `scalyr_kv_renamed/`?

Comment (pheanex) [reply kube_log_watcher/templates/scalyr.json.jinja2]: Won't we have to use some regex like this (^application=.*/component=.*/version=.*$) to detect this special format anyway?
So the format acts as some kind of magic token already?

File: delivery.yaml (hunk 1 above)
Comment (pheanex) [reply delivery.yaml]: Reverted to python3.5

File: kube_log_watcher/templates/scalyr.json.jinja2 (hunk 2 above)
Comment (anton-ryzhov) [reply kube_log_watcher/templates/scalyr.json.jinja2]: I'm for using more flexible format and parser. What if we'll decide to add or remove some part from this filename.
I thought to implement it as "if path is start with special prefix, split items by `/` and keys-values by `=`". Without any bind to property names.

Comment (pheanex) [reply kube_log_watcher/templates/scalyr.json.jinja2]: As discussed we will use question mark as prefix (similar to query-string in URL)

Comment (anton-ryzhov) [reply kube_log_watcher/templates/scalyr.json.jinja2]: Don't you want to use http://jinja.pocoo.org/docs/2.10/templates/#urlencode ?

**Comment by pheanex** [comment]
:+1:

File: kube_log_watcher/templates/scalyr.json.jinja2 (hunk 2 above)
Comment (anton-ryzhov) [reply kube_log_watcher/templates/scalyr.json.jinja2]: Discussed and decided to not using this to keep defined parameters order

**Comment by anton-ryzhov** [comment]
:+1:

**Comment by pheanex** [comment]
:+1:

**Pull request merged**